src/snapshot/svgtoimg.js

```diff
@@ -9,62 +9,61 @@
 
 'use strict';
 
-var EventEmitter = require('events').EventEmitter;
-
 function svgToImg(opts) {
```

Comment:
ok, I'll get that working.  Maybe in `2.0.0` we can move to all promises.  Are you thinking in `opts` something like `{promise:true}`?